core/src/main/java/org/apache/accumulo/core/cryptoImpl/AESCryptoService.java

```diff
@@ -133,7 +133,7 @@ public FileDecrypter getFileDecrypter(CryptoEnvironment environment) {
   }
 
   private static boolean checkNoCrypto(byte[] params) {
-    byte[] noCryptoBytes = NO_CRYPTO_VERSION.getBytes(Charset.forName("UTF-8"));
+    byte[] noCryptoBytes = NO_CRYPTO_VERSION.getBytes(StandardCharsets.UTF_8);
```

Comment:
Many places in the codebase that use this constant statically import it.  Like 

https://github.com/apache/accumulo/blob/3c37cc97c12cc3abeb27882fa2f79a64ca32a9f9/core/src/main/java/org/apache/accumulo/core/iteratorsImpl/conf/ColumnSet.java#L19

It would be nice to do that for consistency with other code.

Reply:
Thanks for reviewing @keith-turner. There were some files with this pattern of not defining it as static too. To bring in complete consistency, changing those files too to match your above suggestion of defining the constant as static import.